cosmic-comp-config/src/lib.rs

```diff
@@ -54,6 +54,8 @@ pub struct XkbConfig {
     pub layout: String,
     pub variant: String,
     pub options: Option<String>,
+    pub repeat_delay: u32,
+    pub repeat_rate: u32,
```

Comment:
This will reset existing config files, that don't contain these values yet, which I would like to avoid. Can you add `#[serde(default = ..)]` tags please?

(Those take a function, so you need to add `default_repeat_delay` and `default_repeat_rate` functions as well. (And best use them in the default implementation as well, so that these values are only declared once. See https://serde.rs/field-attrs.html#default--path)

Reply:
Done!